wolfcrypt/src/random.c

```diff
@@ -1671,20 +1678,27 @@ static int _InitRng(WC_RNG* rng, byte* nonce, word32 nonceSz,
                     ret = DRBG_FAILURE;
                 }
             }
-#else
+    #else
```

Comment:
These aren't aligned with #ifdef???

Reply:
I believe I fixed this one. Also added some whitespace around `CUSTOM_RAND_GENERATE_BLOCK` for clarity.